streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -520,6 +568,8 @@ private Stream<Task> getStreamOfTasks() {
                         restoredActiveTasks.stream(),
                         Stream.concat(
                             exceptionsAndFailedTasks.stream().flatMap(exceptionAndTasks -> exceptionAndTasks.getTasks().stream()),
-                            removedTasks.stream()))));
+                            Stream.concat(
```

Comment:
Could you please add a unit test `shouldGetTasksFromPausedTasks()`?

Reply:
ack.